rules_building_block/lateral_movement_remote_services_ssh.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2023/07/10"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/07/10"
+
+[rule]
+author = ["Elastic"]
+building_block_type = "default"
+description = """
+Adversaries may use Valid Accounts to log into remote machines using Secure Shell (SSH).
+The adversary may then perform actions as the logged-on user.
+"""
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Monitor Remote SSH Login"
+references = ["https://attack.mitre.org/techniques/T1021/004/"]
+risk_score = 47
+rule_id = "1b691b4b-e99b-400b-a573-358d30dad200"
+severity = "medium"
+tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Lateral Movement", "Rule Type: BBR"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and event.action == "exec" and process.name : ("sshd", "ssh")
```

Comment:
Do we want to see every instance of SSH being launched, which is what we monitor here, or do we want to monitor remote SSH logins? Because the latter might better be done through authentication logs.

Reply:
Discussed in RAD. removing this noisy ones